Sources/TuistGenerator/Extensions/Graph+Extras.swift

```diff
@@ -30,12 +31,30 @@ extension TuistGraph.Graph {
                 return false
             }
 
+            for excludeTargetString in excludeTargetsContaining {
+                if target.target.name.lowercased().contains(excludeTargetString.lowercased()) {
+                    return false
+                }
+            }
```

Comment:
Maybe we can accept a regex here to be more flexible (@pepicrft @fortmarek )

Reply:
I've streamlined the code and integrated regex for target and dependency name filtering, as reflected in commit 35e3731.